### DISCUSSION TITLE
샌드박스 활성화가 안되는 것 같습니다.

### DISCUSSION THREAD

**Discussion by Dwarveney** [devrel-kr/HackaLearn#129] [2 Q&A - 클라우드 스킬 챌린지] [answered]
azure 정적 웹앱 만들기 실습 중입니다. 
마이크로소프트/outlook 다 로그인되고 이메일 등록하고 azure에서 카드인증도 했는데 계속해서 같은 메시지가 뜹니다.
현재 엣지 Inprivate 모드인데 시도해볼 만한 다른 방법이 없을까요?
(![질문](https://user-images.githubusercontent.com/88477651/128807302-bf116cca-7678-48d3-882a-b8d9c079b3df.JPG))

**Comment by jung-youjin** [comment]
@Dwarveney 저도 비슷하게 새로운 계정 등록 후 인증 다 마치고 나서도 활성화가 안된다는 비슷한 메시지를 접했었는데요, 저 같은 경우는 하루 이틀 정도 지나고 다시 확인하니 잘 되더군요... 이유는 잘 모르겠어서 도움이 안 될 것 같네요 ㅜㅜ @justinyoo 호옥시 해결 방법 아시나요?!?! 🙏🏼

**Comment by justinyoo** [comment, marked as answer]
Microsoft Learn 프로필과 Microsoft 계정이 연결이 안됐거나 할 때 생기는 문제 같아요. 혹시 [문제해결](https://docs.microsoft.com/ko-kr/learn/support/troubleshooting?WT.mc_id=devcloud-33673-juyoo&ocid=AID3035096) 링크에서 비슷한 상황을 찾아 보셨나요?

* `“onmicrosoft.com” 계정을 사용하여 샌드박스를 활성화할 수 없습니다.`
* `개인 또는 회사 계정 연결`

이 두 가지 상황과 비슷한 것 같은데, 한 번 시도해 보시겠어요? 만약 이렇게 해도 안된다면 새로 생성하신 무료 구독을 통해 진행하시기를 추천 드려요.
